packages/sdk-core/src/constants.ts

```diff
@@ -32,29 +31,41 @@ const subgraphReleaseTag =
     (globalThis.process && globalThis.process.env.SUBGRAPH_RELEASE_TAG) ||
     DefaultSubgraphReleaseTag.value;
 
-const getResolverData = (chainId: number): IResolverData => {
+export const EMPTY_NETWORK_DATA = {
+    subgraphAPIEndpoint: "",
+    networkName: "",
+    nativeTokenSymbol: "",
+    addresses: {
+        resolver: "",
+        host: "",
+        governance: "",
+        cfaV1: "",
+        cfaV1Forwarder: "",
+        idaV1: "",
+        superTokenFactory: "",
+        superfluidLoader: "",
+        toga: "",
+    },
+};
+export type NetworkData = typeof EMPTY_NETWORK_DATA;
+
+const getAddressesData = (chainId: number): NetworkData => {
     const networkData = metadata.networks.find((x) => x.chainId === chainId);
-    if (!networkData)
-        return {
-            subgraphAPIEndpoint: "",
-            networkName: "",
-            resolverAddress: "",
-            nativeTokenSymbol: "",
-        };
+    if (!networkData) return EMPTY_NETWORK_DATA;
```

Comment:
Why not just return `null` or `undefined`? It's better to be explicit than have empty strings and weird empty objects be passed around...

Reply:
hmmm touché